openshift/WFLY-12904.adoc

```diff
@@ -0,0 +1,154 @@
+= Quickstart for EJB remoting and transaction recovery on OpenShift
+:author:            Ondrej Chaloupka
+:email:             ochaloup@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          openshift,transactions,EJB,remoting,recovery
+
+
+== Overview
+
+This is a follow-up task to https://issues.jboss.org/browse/CLOUD-2262[CLOUD-2262] where the goal was to provide
+a safe way to run EJB remote calls amongst WFLY servers on OpenShift and ensure safe transaction recovery
+for propagated XA transactions.
+
+The goal of this proposal is to document this effort with a small working application
+that could be used as a reference when a developer needs to set up all the parts correctly.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-12904[WFLY-12909]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/CLOUD-2262[CLOUD-2262]
+
+=== Dev Contacts
+
+* mailto:{ochaloup@redhat.com}[{author}]
+
+=== QE Contacts
+
+* mailto:{msimka@redhat.com}[msimka@redhat.com]
+
+=== Testing By
+
+[x] Engineering
+[ ] QE
+
+=== Affected Projects or Components
+
+* WildFly quickstarts
+
+=== Other Interested Projects
+
+* Transactions
+* EJB
+
+== Description of the quickstart
+
+The goal of the quickstart is to provide two simple EJB based applications
+which communicates with each other over EJB remoting.
+The first one will be placed on the first WildFly "client server"
+and is accessed over REST to invoke some operation.
+The second one will be deployed on the second server
+and will be called over EJB to process some a work.
+
+Both EJBs (on "client server" and on the second server)
+need to process some "transactional" work (which is e.g. inserting data to a database)
+for transaction manager may run XA transactions.
+
+The way of running the transaction work is about to be investigated.
+As the goal is that this quickstart would be run not only on standalone servers
+but mainly then focused for OpenShift showcase the type of resource
+(database, JMS, mock XA resource) is important for having an easy way how to
+get it to the OpenShift docker image.
```

Comment:
Could you please reword this sentence?

Reply:
Thanks for the feedback. Please, check if it's better now.

Reply:
looks good, thanks